src/interface/meshblock_data.hpp

```diff
@@ -258,6 +261,22 @@ class MeshBlockData {
     return GetAllVariables(sparse_ids, flux).unique_ids();
   }
 
+  // Queries related to swarm data
+  DataCollection<SwarmContainer> &GetSwarmData(int n = 0) {
```

Comment:
Fail on `n > 0`? It looks like we rely downstream on only using the `Get()` method to get the default stage from `DataCollection<SwarmContainer>`. Does anything go wrong if we just get rid of the `DataCollection` and store a `SwarmContainer` directly (and still fail here for `n != 0`)?

Reply:
Actually I'm a bit confused. What does `n` signify here? Is there a reason to allow us to pass in `n` at all? I'd rather fail at compile time than run time.

Reply:
Resolved via comment elsewhere.  Addressed your concern @brryan.